### ISSUE TITLE
Build Error: Node Sass does not yet support current environment: OS X 64-bit with Unsupported runtime (57)

### ISSUE THREAD

**Issue by adrianbrink** [ethereumproject/emerald-wallet#275]
After installing `emerald-rs` with `cargo install` I ran `npm run build:all` inside the `emerald-wallet` directory (I ran `npm install` beforehand).

Apparently the build fails on newest macOS (10.12.5) with node (v8.1.2) and npm (4.6.1).

Any help would be appreciated.

The output is this: 
```
➜  emerald-wallet git:(master) ✗ npm run build:all

> EmeraldWallet@0.2.0 build:all /Users/adrian/projects/ethereumproject/emerald-wallet
> node build-electron & node build-web --for-electron

Hash: 0d822cb699d0bf7e6848
Time: 5296ms
          Asset     Size  Chunks                    Chunk Names
webpack-main.js  1.74 MB       0  [emitted]  [big]  main

WARNING in ./~/bn/lib/wrap.js
6:11-30 Critical dependency: the request of a dependency is an expression

WARNING in ./~/keybase-nacl/lib/main.js
16:13-32 Critical dependency: the request of a dependency is an expression

WARNING in ./~/encoding/lib/iconv-loader.js
9:12-34 Critical dependency: the request of a dependency is an expression
(node:12718) DeprecationWarning: loaderUtils.parseQuery() received a non-string value which can be problematic, see https://github.com/webpack/loader-utils/issues/56
parseQuery() will be replaced with getOptions() in the next major version of loader-utils.
Hash: 627c95cc34a70b92f9c5
Time: 16908ms
                        Asset       Size  Chunks                    Chunk Names
fonts/poppins-latin-500.woff2    13.9 kB          [emitted]
  fonts/poppins-latin-300.eot    15.3 kB          [emitted]
  fonts/poppins-latin-500.eot    15.8 kB          [emitted]
  fonts/poppins-latin-600.eot    15.7 kB          [emitted]
  fonts/poppins-latin-700.eot    15.3 kB          [emitted]
  fonts/poppins-latin-300.svg    39.6 kB          [emitted]
  fonts/poppins-latin-400.svg    39.7 kB          [emitted]
  fonts/poppins-latin-500.svg    39.6 kB          [emitted]
  fonts/poppins-latin-600.svg      39 kB          [emitted]
  fonts/poppins-latin-700.svg    39.4 kB          [emitted]
 fonts/poppins-latin-300.woff    17.5 kB          [emitted]
fonts/poppins-latin-300.woff2    13.5 kB          [emitted]
 fonts/poppins-latin-400.woff    17.4 kB          [emitted]
fonts/poppins-latin-400.woff2    13.5 kB          [emitted]
 fonts/poppins-latin-500.woff    17.9 kB          [emitted]
  fonts/poppins-latin-400.eot    15.3 kB          [emitted]
 fonts/poppins-latin-600.woff    17.7 kB          [emitted]
fonts/poppins-latin-600.woff2    13.8 kB          [emitted]
 fonts/poppins-latin-700.woff    17.5 kB          [emitted]
fonts/poppins-latin-700.woff2    13.5 kB          [emitted]
          images/etc_logo.png    7.94 kB          [emitted]
       images/ether-title.png    15.9 kB          [emitted]
           images/loading.gif    44.7 kB          [emitted]
            images/logo-1.png    23.1 kB          [emitted]
            images/logo-2.png    3.65 kB          [emitted]
          images/notes-bg.png  629 bytes          [emitted]
                     tests.js    15.4 kB       0  [emitted]         tests
                     index.js    3.92 MB       1  [emitted]  [big]  index
                    index.css    21.5 kB       1  [emitted]         index
                   index.html  304 bytes          [emitted]

WARNING in ./~/bindings/bindings.js
76:22-40 Critical dependency: the request of a dependency is an expression

WARNING in ./~/bindings/bindings.js
76:43-53 Critical dependency: the request of a dependency is an expression

WARNING in ./~/encoding/lib/iconv-loader.js
9:12-34 Critical dependency: the request of a dependency is an expression

ERROR in ./~/css-loader!./~/sass-loader/lib/loader.js?{"includePaths":["/Users/adrian/projects/ethereumproject/emerald-wallet/~/compass-mixins/lib"]}!./src/index.scss
Module build failed: Error: Node Sass does not yet support your current environment: OS X 64-bit with Unsupported runtime (57)
For more information on which environments are supported please see:
https://github.com/sass/node-sass/releases/tag/v4.5.2
    at module.exports (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/node-sass/lib/binding.js:13:13)
    at Object.<anonymous> (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/node-sass/lib/index.js:14:35)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/sass-loader/lib/loader.js:3:14)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
 @ ./src/index.scss 4:14-127
 @ ./src/index.js

ERROR in ./~/css-loader!./~/sass-loader/lib/loader.js?{"includePaths":["/Users/adrian/projects/ethereumproject/emerald-wallet/~/compass-mixins/lib"]}!./~/font-awesome/scss/font-awesome.scss
Module build failed: Error: Node Sass does not yet support your current environment: OS X 64-bit with Unsupported runtime (57)
For more information on which environments are supported please see:
https://github.com/sass/node-sass/releases/tag/v4.5.2
    at module.exports (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/node-sass/lib/binding.js:13:13)
    at Object.<anonymous> (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/node-sass/lib/index.js:14:35)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/sass-loader/lib/loader.js:3:14)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
 @ ./~/font-awesome/scss/font-awesome.scss 4:14-114
 @ ./src/index.js

ERROR in ./~/css-loader!./~/sass-loader/lib/loader.js?{"includePaths":["/Users/adrian/projects/ethereumproject/emerald-wallet/~/compass-mixins/lib"]}!./src/components/main.scss
Module build failed: Error: Node Sass does not yet support your current environment: OS X 64-bit with Unsupported runtime (57)
For more information on which environments are supported please see:
https://github.com/sass/node-sass/releases/tag/v4.5.2
    at module.exports (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/node-sass/lib/binding.js:13:13)
    at Object.<anonymous> (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/node-sass/lib/index.js:14:35)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/sass-loader/lib/loader.js:3:14)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
 @ ./src/components/main.scss 4:14-132
 @ ./src/components/main.js
 @ ./src/index.js

ERROR in ./~/css-loader!./~/sass-loader/lib/loader.js?{"includePaths":["/Users/adrian/projects/ethereumproject/emerald-wallet/~/compass-mixins/lib"]}!./~/roboto-fontface/css/roboto/sass/roboto-fontface.scss
Module build failed: Error: Node Sass does not yet support your current environment: OS X 64-bit with Unsupported runtime (57)
For more information on which environments are supported please see:
https://github.com/sass/node-sass/releases/tag/v4.5.2
    at module.exports (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/node-sass/lib/binding.js:13:13)
    at Object.<anonymous> (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/node-sass/lib/index.js:14:35)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/sass-loader/lib/loader.js:3:14)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
 @ ./~/roboto-fontface/css/roboto/sass/roboto-fontface.scss 4:14-129
 @ ./src/index.js

ERROR in ./~/font-awesome/scss/font-awesome.scss
Module build failed: ModuleBuildError: Module build failed: Error: Node Sass does not yet support your current environment: OS X 64-bit with Unsupported runtime (57)
For more information on which environments are supported please see:
https://github.com/sass/node-sass/releases/tag/v4.5.2
    at module.exports (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/node-sass/lib/binding.js:13:13)
    at Object.<anonymous> (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/node-sass/lib/index.js:14:35)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/sass-loader/lib/loader.js:3:14)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at runLoaders (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/NormalModule.js:192:19)
    at /Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:364:11
    at /Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:170:18
    at loadLoader (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/loadLoader.js:27:11)
    at iteratePitchingLoaders (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:169:2)
    at iteratePitchingLoaders (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:165:10)
    at /Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:173:18
    at loadLoader (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/loadLoader.js:36:3)
    at iteratePitchingLoaders (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:169:2)
    at runLoaders (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:362:2)
    at NormalModule.doBuild (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/NormalModule.js:179:3)
    at NormalModule.build (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/NormalModule.js:268:15)
    at Compilation.buildModule (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/Compilation.js:146:10)
    at moduleFactory.create (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/Compilation.js:433:9)
    at factory (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/NormalModuleFactory.js:253:5)
    at applyPluginsAsyncWaterfall (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/NormalModuleFactory.js:99:14)

ERROR in ./src/index.scss
Module build failed: ModuleBuildError: Module build failed: Error: Node Sass does not yet support your current environment: OS X 64-bit with Unsupported runtime (57)
For more information on which environments are supported please see:
https://github.com/sass/node-sass/releases/tag/v4.5.2
    at module.exports (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/node-sass/lib/binding.js:13:13)
    at Object.<anonymous> (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/node-sass/lib/index.js:14:35)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/sass-loader/lib/loader.js:3:14)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at runLoaders (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/NormalModule.js:192:19)
    at /Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:364:11
    at /Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:170:18
    at loadLoader (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/loadLoader.js:27:11)
    at iteratePitchingLoaders (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:169:2)
    at iteratePitchingLoaders (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:165:10)
    at /Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:173:18
    at loadLoader (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/loadLoader.js:36:3)
    at iteratePitchingLoaders (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:169:2)
    at runLoaders (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:362:2)
    at NormalModule.doBuild (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/NormalModule.js:179:3)
    at NormalModule.build (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/NormalModule.js:268:15)
    at Compilation.buildModule (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/Compilation.js:146:10)
    at moduleFactory.create (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/Compilation.js:433:9)
    at factory (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/NormalModuleFactory.js:253:5)
    at applyPluginsAsyncWaterfall (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/NormalModuleFactory.js:99:14)

ERROR in /Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/extract-text-webpack-plugin/loader.js??ref--1-0!/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/style-loader/index.js!/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/css-loader/index.js!/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/sass-loader/lib/loader.js??ref--1-3!/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/font-awesome/scss/font-awesome.scss doesn't export content

ERROR in /Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/extract-text-webpack-plugin/loader.js??ref--1-0!/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/style-loader/index.js!/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/css-loader/index.js!/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/sass-loader/lib/loader.js??ref--1-3!/Users/adrian/projects/ethereumproject/emerald-wallet/src/index.scss doesn't export content

ERROR in ./src/components/main.scss
Module build failed: ModuleBuildError: Module build failed: Error: Node Sass does not yet support your current environment: OS X 64-bit with Unsupported runtime (57)
For more information on which environments are supported please see:
https://github.com/sass/node-sass/releases/tag/v4.5.2
    at module.exports (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/node-sass/lib/binding.js:13:13)
    at Object.<anonymous> (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/node-sass/lib/index.js:14:35)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/sass-loader/lib/loader.js:3:14)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at runLoaders (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/NormalModule.js:192:19)
    at /Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:364:11
    at /Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:170:18
    at loadLoader (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/loadLoader.js:27:11)
    at iteratePitchingLoaders (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:169:2)
    at iteratePitchingLoaders (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:165:10)
    at /Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:173:18
    at loadLoader (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/loadLoader.js:36:3)
    at iteratePitchingLoaders (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:169:2)
    at runLoaders (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:362:2)
    at NormalModule.doBuild (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/NormalModule.js:179:3)
    at NormalModule.build (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/NormalModule.js:268:15)
    at Compilation.buildModule (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/Compilation.js:146:10)
    at moduleFactory.create (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/Compilation.js:433:9)
    at factory (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/NormalModuleFactory.js:253:5)
    at applyPluginsAsyncWaterfall (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/NormalModuleFactory.js:99:14)

ERROR in /Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/extract-text-webpack-plugin/loader.js??ref--1-0!/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/style-loader/index.js!/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/css-loader/index.js!/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/sass-loader/lib/loader.js??ref--1-3!/Users/adrian/projects/ethereumproject/emerald-wallet/src/components/main.scss doesn't export content

ERROR in ./~/roboto-fontface/css/roboto/sass/roboto-fontface.scss
Module build failed: ModuleBuildError: Module build failed: Error: Node Sass does not yet support your current environment: OS X 64-bit with Unsupported runtime (57)
For more information on which environments are supported please see:
https://github.com/sass/node-sass/releases/tag/v4.5.2
    at module.exports (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/node-sass/lib/binding.js:13:13)
    at Object.<anonymous> (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/node-sass/lib/index.js:14:35)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/sass-loader/lib/loader.js:3:14)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at runLoaders (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/NormalModule.js:192:19)
    at /Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:364:11
    at /Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:170:18
    at loadLoader (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/loadLoader.js:27:11)
    at iteratePitchingLoaders (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:169:2)
    at iteratePitchingLoaders (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:165:10)
    at /Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:173:18
    at loadLoader (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/loadLoader.js:36:3)
    at iteratePitchingLoaders (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:169:2)
    at runLoaders (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/loader-runner/lib/LoaderRunner.js:362:2)
    at NormalModule.doBuild (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/NormalModule.js:179:3)
    at NormalModule.build (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/NormalModule.js:268:15)
    at Compilation.buildModule (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/Compilation.js:146:10)
    at moduleFactory.create (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/Compilation.js:433:9)
    at factory (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/NormalModuleFactory.js:253:5)
    at applyPluginsAsyncWaterfall (/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/webpack/lib/NormalModuleFactory.js:99:14)

ERROR in /Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/extract-text-webpack-plugin/loader.js??ref--1-0!/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/style-loader/index.js!/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/css-loader/index.js!/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/sass-loader/lib/loader.js??ref--1-3!/Users/adrian/projects/ethereumproject/emerald-wallet/node_modules/roboto-fontface/css/roboto/sass/roboto-fontface.scss doesn't export content
```

**Comment by splix** [comment]
@adrianbrink can you try with node 6? we had similar problem with node-sass and node 8 on windows, so switched to node 6 for builds

**Comment by adrianbrink** [comment]
It fixes the problem.

I've opened a PR to highlight that we need node 6.

**Comment by dulanov** [comment]
Fixed by #276
